components/CodeBlock.tsx

```diff
@@ -45,164 +64,94 @@ export interface CodeBlockProps {
 export function RawCodeBlock({
   code,
   language,
-  className: extraClassName,
+  class: extraClassName,
```

Comment:
It'd be preferable to avoid keywords like `class` ... is there a reason for this change?

Reply:
we declare classes with `class` attribute, this is just to keep in line with that